packages/nx-playwright/src/executors/playwright-executor/schema-types.d.ts

```diff
@@ -12,7 +12,7 @@ export interface PlaywrightExecutorSchema {
   reporter?: string;
   browser?: 'chromium' | 'firefox' | 'webkit' | 'all';
   packageRunner?: PackageRunner;
-  proj?: string;
+  testProject?: string;
```

Comment:
In my PR #76 I also added `--proj` to `README.md` so I guess it needs updating there, too :)